pydicom/valuerep.py

```diff
@@ -646,9 +646,13 @@ def __init__(self, val, encodings=None, original_string=None):
             # this is the decoded string - save the original string if
             # available for easier writing back
             self.original_string = original_string
-            self._components = tuple(val.split('='))
+            components = val.split('=')
+            # Remove empty elements from the end to avoid trailing '='
+            while len(components) and not components[-1]:
+                components.pop()
+            self._components = tuple(components)
```

Comment:
Note: I changed this to conform to the previous behavior and to `PersonNameUnicode`. The trailing equal signs are removed anyway during encoding later, and this way, the json encoding behaves the same in Python 2 and 3.